src/components/Contentful/Grid.js

```diff
@@ -35,19 +35,78 @@ function GridContainer({
   )
 }
 
-function GridCol({ children, columnWidth, columnOffset }) {
-  columnWidth = columnWidth || 12
-  columnOffset = columnOffset || 0
+function GridCol({
+  children,
+  extraLargeColumnWidth,
+  extraLargeColumnOffset,
+  largeColumnWidth,
+  largeColumnOffset,
+  mediumColumnWidth,
+  mediumColumnOffset,
+  smallColumnWidth,
+  smallColumnOffset,
+  extraSmallColumnWidth,
+  extraSmallColumnOffset,
+}) {
+  let colClasses = 'my-2'
+
+  if (extraLargeColumnWidth !== null) {
```

Comment:
This should probably be `if (extraLargeColumnWidth)` rather than `if(extraLargeColumnWidth !== null)` - a bit shorter and covers undefined as well as null.

Alternatively can inline it as `extraLargeColumnWidth && colClasses += ' col-xl-${extraLargeColumnWidth}'` if you're feeling fancy

Reply:
Nah that wont work as it'll ignore it if extraLargeColumnWidth === 0. And we need 0!

Reply:
Oh yeah, good point! Didn't register that it was a numerical value